inprocess/src/main/java/io/grpc/inprocess/AnonymousInProcessSocketAddress.java

```
@@ -34,7 +34,7 @@ public final class AnonymousInProcessSocketAddress extends SocketAddress {
 
   @Nullable
   @GuardedBy("this")
-  private InProcessServer server;
+  private transient InProcessServer server;
```

Serializing this object won't work. I'd be tempted to make it fail explicitly.